suite/tests/api/ir_riscv64.expect

```diff
@@ -221,6 +221,8 @@ c.xor  a5 -> fp
 c.sub  a5 -> fp
 test_integer_arith complete
 lui    0x2a -> a0
+<Internal Error: Failed to encode instruction: 'auipc  0x0000004000016234 -> a0'
```

Comment:
If this is ignored, why do we need to list it here?

Reply:
It's ignored only in static builds (e.g. `api.ir-static`) not in dynamic builds (e.g. `api.ir`).